ddsrouter_core/src/cpp/core/DDSRouterImpl.cpp

```diff
@@ -331,6 +345,14 @@ void DDSRouterImpl::discovered_topic_(
     }
 }
 
+void DDSRouterImpl::discovered_endpoint_(
+        const Endpoint& endpoint) noexcept
+{
+    logDebug(DDSROUTER, "Discovered endpoint: " << endpoint << ".");
```

Comment:
This log refers to DDS Router behaviour, so it should specify that this is happening in DDS Router. If you want to log something in DiscoveryDatabase, do it in that class implementation.